spec/carrier_metadata_spec.rb

```diff
@@ -19,7 +19,7 @@
       carrier = 'usps'
       types = %w[service_levels predefined_packages]
 
-      metadata = client.beta_carrier_metadata.retrieve_carrier_metadata(['usps'], types)
+      metadata = client.carrier_metadata.retrieve_carrier_metadata(['usps'], types)
```

Comment:
Ah, nice catch!!

Reply:
Actually, I just realized all the libs except C# are called `.retrieve_carrier_metadata()`, but since we never release this beta feature in Ruby, I will keep it as `retrieve()`

Reply:
Yup, another minor breaking change for all libs😅